helm/templates/deployment.yml

```diff
@@ -107,6 +107,16 @@ spec:
                 secretKeyRef:
                   name: nginxs3proxy
                   key: proxycachevalidforbidden
+            - name: PROVIDE_INDEX_PAGE
```

Comment:
Do we need to add these as secrets or could they just be hardcoded?
They don't feel sensitive

Reply:
Good point - you're right, some are not sensitive. 

It's just that I prefer not to reveal the application configuration.

Reply:
GDS guidelines are challenging us to be as open as possible including our configuration
https://www.gov.uk/service-manual/technology/making-source-code-open-and-reusable#making-configuration-code-open
Do you think this is something we can do or is there a specific concern that we can address?

Reply:
No concerns - happy to get over my personal preference!

Presume just the s3 access key and secret key would be kept as secrets?

Reply:
Interesting. I never really spotted how much was already in secrets because I only looked at the diff in the PR.

I would also keep bucket name secret too.
[Amazon S3 Preventative Security Best Practices](https://docs.aws.amazon.com/AmazonS3/latest/userguide/security-best-practices.html#:~:text=Unless%20you%20explicitly%20require%20anyone,S3%20bucket%20is%20not%20public).

So
S3_BUCKET_NAME
S3_ACCESS_KEY_ID
S3_SECRET_KEY

Reply:
Updated the configuration as requested